src/Polynomials/MonomialBases.jl

```diff
@@ -180,7 +182,7 @@ function evaluate!(
   setsize!(g,(D,n))
   for i in 1:np
     @inbounds xi = x[i]
-    _gradient_nd!(v,xi,f.orders,f.terms,c,g,T)
+    _gradient_nd!(v,xi,f.orders,f.terms,c,g,T,Val(isbits(eltype(c))))
```

Comment:
`Val(isbits(eltype(c))` this call is loop invariant, you always build the same object for every i. The compiler might be able to realize it and do it automatically, but better to extract the call out of the loop to be on the safe side.

Reply:
done

Reply:
Great, thanks!